packages/did/src/DidBatcher/DidBatchBuilder.ts

```diff
@@ -61,7 +61,7 @@ export class DidBatchBuilder {
   /**
    * Create a new builder with the provided WS connection and [[FullDidDetails]].
    *
-   * @param api The [[ApiPromise]] wrapping the WS connection to a KILT RPC node.
+   * @param api The ApiPromise wrapping the WS connection to a KILT RPC node.
```

Comment:
Does this give a reference error? Or did you think it's simply not needed?

Reply:
Since `ApiPromise` is a polkadot type and this documentation only builds entries for things declared in the `@kiltprotocol` namespace, it can't reference it.